frontend/src/pages/UserDashboard/UserDashboard.module.css

```diff
@@ -0,0 +1,197 @@
+.dashboardContainer {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  background-color: #f4f4f4;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  background-color: #09245a;
```

Comment:
We need to use similar colors. In Figma designs for the teacher and user this color was used #122A50